src/App.tsx

```diff
@@ -1,26 +1,220 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { FC, useEffect, useRef, useState } from 'react';
+
+import { USER_ID, todoService } from './api/todos';
 import { UserWarning } from './UserWarning';
+import Footer from './components/Footer';
+import Header from './components/Header';
+import TodoList from './components/TodoList';
+
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
+import { Errors } from './types/Errors';
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [title, setTitle] = useState('');
```

Comment:
[Optional fix] If you keep this state on the global level, then the whole app rerenders on each new character entered into the header

Please move it to the header

Anyways, you are using the title only inside `addTodo` func, so you can just make it an argument of the `addTodo`

Reply:
Hello! At first my function looked just like you say, but then I realized I need to accept the event to do event.preventDefault() in addTodo func, so changed everything. Can I do it without event.preventDefault?

Reply:
Hello! I cannot move title into the header, because I need to setTitle to '' in my addTodo func. And I cannot move my addTodo func into the header because I need to setErrorMessage in it, and my errorMessage needs to be on top to be accessible everywhere> The better solution would be to make global context for the errorMessage, but here I didnt do it :( 
But I changed everything else that you said, thank you!